templates/_devcontainer/iobroker/_Dockerfile.ts

```diff
@@ -8,6 +8,9 @@ const templateFunction: TemplateFunction = answers => {
 	const template = `
 FROM iobroker/iobroker:latest
 RUN ln -s /opt/iobroker/node_modules/ /root/.node_modules
+# Needed to use git-ssh in devcontainer
+RUN apt-get update &&\\
+    apt-get -y --no-install-recommends install openssh-client
```

Comment:
Formatting here seems to mix tabs and spaces, would be great to have this consistent (tabs):
![grafik](https://github.com/user-attachments/assets/9df8139e-a18b-4c0c-90c6-a9d7fc83ed87)

Reply:
Whats the fundamental process here? Tabs are used for the code in the create-adapter projects. Thats fine - but the indentation within the template should follow the answer the user gives for `indendation`, shouldn't it? Currently the templates use a mix of spaces and tabs depending which file is in focus. I changed it to tabs no bc. of the review comment but if a user wants to have spaces, will they get a issue?